src/main/java/net/neoforged/moddevgradle/internal/WorkflowArtifact.java

```diff
@@ -0,0 +1,16 @@
+package net.neoforged.moddevgradle.internal;
+
+import org.jetbrains.annotations.ApiStatus;
+
+@ApiStatus.Internal
+public enum WorkflowArtifact {
+    COMPILED(""),
+    COMPILED_WITH_SOURCES("-merged"),
+    SOURCES("-sources"),
+    CLIENT_RESOURCES("-client-extra");
```

Comment:
Don't you want to call it something a bit nicer? E.g. `-client-resources-aka-client-extra`? 😄